src/question.ts

```diff
@@ -101,6 +101,7 @@ export class Question extends SurveyElement<Question>
   protected setIsMobile(val: boolean) { }
 
   @property({ defaultValue: false, onSet: (val: boolean, target: Question) => {
+    target.renderMinWidth = !val;
```

Comment:
Do we have renderMinWidth property?

Reply:
Yep, I added it to SurveyElement class 
https://github.com/surveyjs/survey-library/pull/6907/files#diff-d042e63e655cfdcd2783a19d550645690de316e2e69aca982f9b52c30d5ad393R910